content/SCALE/SCALETutorials/Shares/MixedModeShares.md

```diff
@@ -0,0 +1,225 @@
+#### Starting the Service Using the Windows SMB Share
+
+From the **Sharing** screen, click on the **Windows (SMB) Shares** <span class="material-icons">more_vert</span> to display the service options, which are **Turn Off Service** if the service is running or **Turn On Service** if the service is not running.
+
+{{< trueimage src="/images/SCALE/Shares/SharingSMBServicesActionOptions.png" alt="SMB Service Options" id="SMB Service Options" >}}
+
+Each SMB share on the list also has a toggle to enable or disable the service for that share.
+
+#### Starting the Service Using System Settings
+
+Go to **System Settings > Services** and click the toggle for **SMB**.
+Set **Start Automatically** if you want the service to activate when TrueNAS boots.
+{{< /expand >}}
+
+### Configuring the SMB Service
+
+Configure the SMB service by clicking **Config Service** from the <span class="material-icons">more_vert</span> dropdown menu on the **Shares** screen or by clicking <i class="material-icons" aria-hidden="true" title="Configure">edit</i> on the **Services** screen.
+Unless you need a specific setting or are configuring a unique network environment, we recommend using the default settings.
+
+### Starting the NFS Service
+
+You can start the service from the **Unix Shares (NFS)** header on the **Sharing** screen or in **System Settings > Services**.
+
+{{< expand "Click for more information" "v" >}}
+#### Starting the Service Using the Unix Shares (NFS) Share
+
+From the **Sharing** screen, click on the **Unix Shares (NFS)** <span class="material-icons">more_vert</span> to display the service options, which are **Turn Off Service** if the service is running or **Turn On Service** if the service is not running.
+
+{{< trueimage src="/images/SCALE/Shares/NFSWidgetOptions.png" alt="Unix (NFS) Shares Widget Options" id="Unix (NFS) Shares Widget Options" >}}
+
+Each NFS share on the list also has a toggle to enable or disable the service for that share.
+
+#### Starting the Service Using System Settings
+
+Go to **System Settings > Services** and click the toggle for **NFS**.
+Set **Start Automatically** if you want the service to activate when TrueNAS boots.
+
+{{< hint type=note >}}
+The NFS service does not automatically start on boot if all NFS shares are encrypted and locked.
+{{< /hint >}}
+{{< /expand >}}
+
+### Configuring the NFS Service
+
+Configure the NFS service by clicking **Config Service** from the <span class="material-icons">more_vert</span> dropdown menu on the **Shares** screen or by clicking <i class="material-icons" aria-hidden="true" title="Configure">edit</i> on the **Services** screen.
+
+Under **NFSv4**, ensure the **NFSv4** protocol is selected from the **Enabled Protocols** dropdown menu.
+For security hardening, we recommend disabling the **NFSv3** protocol.
+
+Select **Require Kerberos for NFSv4** to enable the Kerberos ticket that generated when you [joined active directory](#joining-active-directory).
```

Comment:
We probably need to add steps regarding how to enable this properly if it involves adding KRB5 SPN as mentioned above.